english/Implication/Implications.txt

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.